exec/vector/src/main/java/org/apache/drill/exec/record/metadata/AbstractColumnMetadata.java

```diff
@@ -295,12 +295,6 @@ public String toString() {
         .toString();
   }
 
-  @JsonProperty("type")
-  @Override
-  public String typeString() {
-    return majorType().toString();
-  }
-
```

Comment:
Please don't remove this method but leave it as abstract with json property annotation:
```
  @JsonProperty("type")
  @Override
  public abstract String typeString();
```